portfolio/src/main/webapp/style.css

```diff
@@ -32,4 +32,7 @@
   color: blanchedalmond;
 }
 
-
+#homelinks img {
```

Comment:
in the above #content, could you also do a minor clean-up of the /*margin-left: auto and margin-right:auto */. github doesn't let me comment on unchanged lines.